app/src/interfaces/map/options.vue

```diff
@@ -56,21 +52,18 @@ export default defineComponent({
 		const { t } = useI18n();
 
 		const isGeometry = props.fieldData.type == 'geometry';
-		const nativeGeometryType = isGeometry ? props.fieldData!.schema!.geometry_type : undefined;
+		const nativeGeometryType = isGeometry ? (props.fieldData!.schema!.geometry_type as GeometryType) : undefined;
 		const compatibleFormat = isGeometry ? ('native' as const) : getGeometryFormatForType(props.fieldData.type);
 
 		const geometryFormat = ref<GeometryFormat>(compatibleFormat!);
-		const geometryType = ref<GeometryType>(
-			geometryFormat.value == 'lnglat' ? 'Point' : nativeGeometryType ?? props.value?.geometryType
-		);
+		const geometryType = ref<GeometryType>(nativeGeometryType ?? 'GeometryCollection');
```

Comment:
If `geometryFormat` is `lnglat`, then `Point` must be set programmatically as it is the only option available and therefore will not be user-editable.  Also, setting the geometryType to `GeometryCollection` will impact the interface behaviour (see other comment)